src/views/PlaygroundView.tsx

```diff
@@ -39,3 +40,26 @@ const LinksContainer = styled.div`
 `
 
 export default PlaygroundView
+
+const VideoMetadata = () => {
```

Comment:
Maybe let's put this in a separate file like the `PlaygroundValidationForm` so it doesn't get messy?